components/main-with-sidebar/example-sidebar-menu/example-sidebar-menu.html

```diff
@@ -0,0 +1,12 @@
+{% extends '@main-with-sidebar--default' %}
+
+{% block content %}
+  <aside class="mzp-l-sidebar">
+    {% render '@sidebar-menu' %}
+  </aside>
+  <main class="mzp-l-main">
+    <p>Main content.</p>
```

Comment:
In the interest of making this a more realistic example we should fill this with more dummy text. A few paragraphs of lorem ipsum, some headings, etc. And we can remove the colored borders so it's a more accurate presentation. Not sure how best to do that with `extends`, maybe just put the styles in a block in the parent template so this one can have an empty block.